.github/workflows/test.yml

```diff
@@ -30,9 +30,9 @@ jobs:
         run: make test
 
   lint:
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-24.04
     steps:
       - uses: actions/checkout@v4
-      - uses: golangci/golangci-lint-action@v6.5.2
+      - uses: golangci/golangci-lint-action@v7
         with:
-          version: v1.62.2
+          version: v2.1
```

Comment:
This maybe?

```suggestion
          version: latest
```

Reply:
It's not good to have it that way, because updates to golangci-lint may well break CI, and we don't want this "automatic breakage" feature.

Reply:
Agree.
Sorry I didn't comment before merging.

Reply:
I understand the logic, thanks for replying me